src/custom/components/AccountDetails/index.tsx

```diff
@@ -148,17 +145,8 @@ export default function AccountDetails({
   const { account, connector, chainId: connectedChainId } = useActiveWeb3React()
   const chainId = supportedChainId(connectedChainId)
   const walletInfo = useWalletInfo()
-  // const theme = useContext(ThemeContext)
-  const dispatch = useDispatch<AppDispatch>()
-
-  const clearAllActivityCallback = useCallback(() => {
-    if (chainId) {
-      batch(() => {
-        dispatch(clearAllTransactions({ chainId }))
-        dispatch(clearOrders({ chainId }))
-      })
-    }
-  }, [dispatch, chainId])
+
+  const explorerOrdersLink = account && connectedChainId && getExplorerAddressLink(connectedChainId, account)
```

Comment:
I fear some merge conflicts with out presign stuff

I'm not sure what will be best to merge first...

Reply:
Oh yeah, there will be conflicts.

I'll merge first =P

After all you need this for your feature